backend/app.py

```diff
@@ -212,7 +212,7 @@ def get_cset_members_items(codeset_ids: List[int] = None, con=CON) -> List[Legac
         csm: false if not in concept set members
     """
     return sql_query(
-        con, f""" 
```

Comment:
FYI: Lots of tiny whitespace like this got removed where it didn't need to be. I don't know how that happened. I would have reverted this, but no time before vacation.